### PR TITLE
Grid clean up 

### PR DESCRIPTION
**Pull request by rouille**
[Pull Request doc](https://breakthrough-energy.github.io/docs/user/git_guide.html#d-pull-request)

### Purpose
Clean up the grid related modules

### What the code is doing
* Read all grid csv files in the `CSVReader` class
* Move the `csv_to_data_frame` function and rename it
* Format docstrings

### Testing
Run existing unit tests

### Where to look
The `AbstractGrid` and the `CSVReader` classes

### Usage Example/Visuals
N/A

### Time estimate
5min

### CODE REVIEW

**Review by BainanXia — APPROVED** [review]
Definitely cleaner. Thanks

File: powersimdata/input/scenario_grid.py [hunk 1]
```diff
@@ -318,6 +318,11 @@ def add_information_to_model(grid):
 
 
 def reindex_model(grid):
+    """Reindex bus id columns.
+
+    :param powersimdata.input.ScenarioGrid grid: grid
```
Comment (jenhagg) [inline powersimdata/input/scenario_grid.py:323]: Think it should be `powersimdata.input.scenario_grid.ScenarioGrid`

Comment (jenhagg) [inline powersimdata/input/scenario_grid.py:323]: Think it should be `powersimdata.input.scenario_grid.ScenarioGrid`

**Review by jenhagg — APPROVED** [review]
Thanks, this makes more sense

**Pull request merged**